contributing-guides/style-guide.md

```diff
@@ -348,6 +348,20 @@ This can be resolved by inserting a comma before the "and" or "or" in the final
 
 > Delete the Git branches, tags, and remotes.
 
+## See also section
+
+- To reference a related command or subcommand, use:
+
+```md
+> See also: `command`.
+```
+
+- To reference related commands or subcommands, use:
+
+```md
+> See also: `command1`, `command2`, `command3`.
+```
+
```

Comment:
Agreed, maybe we can rephrase it a bit (and Markdown highlighting is broken). Will fix it directly in your suggestion.

Reply:
> Agreed, maybe we can rephrase it a bit (and Markdown highlighting is broken). Will fix it directly in your suggestion.

Feel free to do it, I'm not that good at writing English texts 😅

Reply:
Done, feel free to check it out @vitorhcl. Btw, in Markdown when you need to escape highlighting inside a command, we use double backticks at the start and end.

This will fix broken output like `hello `htop`.` -> ``hello `htop`.``

Reply:
LGTM, thanks